Trust and Compliance: Fiscal provisions (#105)
pretix/pretix-docs: docs/trust/fiscal/index.md

# Fiscal requirements

> When you use pretix to sell tickets, you need to make sure that your usage of pretix is in line with tax law.

Reviewer: ```suggestion
If you use pretix to sell any paid goods or services, it is your responsibility to ensure that your usage of pretix is in line with tax law.
```

Tax law differs significantly from country to country.

> Therefore, please select your country in the navigation to receive country-specific information.

Reviewer: ```suggestion
If you need information on the interaction of pretix with tax law in any specific country, click the article for that country in the menu on the left. 
```

> If your country is not contained in the list, it does not mean that you cannot use pretix.

Reviewer: ```suggestion
You can still use pretix if your country is not included in the list. 
```

> Instead, it means that we have not yet been able to compile information on the specific requirements.

Reviewer: ```suggestion
That just means that we have not yet been able to compile information on the specific requirements for that country.
```

> Please reach out to us at [support@pretix.eu](mailto:support@pretix.eu) if we can help you figuring out if pretix can be used in your country.

Reviewer: ```suggestion
If you need help figuring out how to use pretix in your country, reach out to us at [support@pretix.eu](mailto:support@pretix.eu) 
```
